main.py

```diff
@@ -76,16 +85,15 @@ def display_score():
         if game_active:
             if event.type == pygame.KEYDOWN:
                 if event.key == pygame.K_LEFT:
-                    player_speed = -5
+                    accelerating_left = True
```

Comment:
Changed velocity to a boolean representing whether the player is accelerating or not.